packages/xo-server/src/api/network.js

```diff
@@ -12,13 +13,15 @@ export async function create({
   mtu = 1500,
   vlan = 0,
 }) {
-  return this.getXapi(pool).createNetwork({
-    name,
-    description,
-    pifId: pif && this.getObject(pif, 'PIF')._xapiId,
-    mtu: +mtu,
-    vlan: +vlan,
-  })
+  return xapiObjectToXo(
+    await this.getXapi(pool).createNetwork({
+      name,
+      description,
+      pifId: pif && this.getObject(pif, 'PIF')._xapiId,
+      mtu: +mtu,
+      vlan: +vlan,
+    })
+  )
```

Comment:
No other creation methods return an object, they all returns an id, why should this method be different?

Reply:
It returned a XAPI object without ID and in my test i need the network ID

Reply:
So?

Reply:
I can return the ID instead.